python/pyarrow/tests/test_parquet.py

```diff
@@ -2337,7 +2337,22 @@ def _make_example_multifile_dataset(base_path, nfiles=10, file_nrows=5):
 
 
 @pytest.mark.pandas
-def test_ignore_private_directories(tempdir):
+def test_ignore_private_directories_dot(tempdir):
+    dirpath = tempdir / guid()
+    dirpath.mkdir()
+
+    paths = _make_example_multifile_dataset(dirpath, nfiles=10,
+                                            file_nrows=5)
+
+    # private directory
+    (dirpath / '.spark-staging').mkdir()
```

Comment:
Perhaps we can parametrize a single private directory test with a list of different private directories and avoid some code duplication?

Reply:
Yes, this test and the one below can indeed easily be combined